src/MobileUI/Constants.cs

```diff
@@ -20,7 +20,7 @@ public static class Constants
     public const string AuthRedirectUrl = "msauth.com.ssw.consulting://auth";
 
 #if DEBUG
-    public const string AuthorityUri = "https://app-ssw-ident-staging-api.azurewebsites.net";
+    public const string AuthorityUri = "https://identity.ssw.com.au"; // "https://app-ssw-ident-staging-api.azurewebsites.net";
```

Comment:
Should probably revert this before merging

Reply:
Yeah... but now in 95% of cases we use production Identity Server in Debug. I added another line which points to staging and commented it out. 
Is it ok with you?